src/alfasim_score/converter/alfacase/_tests/test_convert_casing.py

```diff
@@ -0,0 +1,102 @@
+from typing import Any
+
+import json
+import pytest
+from pathlib import Path
+from pytest_regressions.data_regression import DataRegressionFixture
+
+from alfasim_score.common import prepare_for_regression
+from alfasim_score.converter.alfacase.convert_alfacase import ScoreAlfacaseConverter
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+
+
+def test_convert_casing_list(
+    data_regression: DataRegressionFixture,
+    score_input_example: ScoreInputReader,
+) -> None:
+    builder = ScoreAlfacaseConverter(score_input_example)
+    casings = builder._convert_casing_list()
+    data_regression.check(
+        [
+            prepare_for_regression(
+                {
+                    "name": casing.name,
+                    "hanger_depth": casing.hanger_depth,
+                    "settings_depth": casing.settings_depth,
+                    "hole_diameter": casing.hole_diameter,
+                    "outer_diameter": casing.outer_diameter,
+                    "inner_diameter": casing.inner_diameter,
+                    "inner_roughness": casing.inner_roughness,
+                    "material": casing.material,
+                    "top_of_filler": casing.top_of_filler,
+                    "filler_material": casing.filler_material,
+                    "material_above_filler": casing.material_above_filler,
+                }
```

Comment:
Since all description classes from alfasim_sdk are attrs, you can use:
```suggestion
                attr.asdict(casing)
```

Reply:
Use this for all other tests

Reply:
Very nice! I'm gonna use this now.

Reply:
All changed.